inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -57,10 +57,13 @@ std::string translate_type_name(const std::string& name) {
     return name;
 }
 
+using const_hash_map_t = std::unordered_map<size_t, int64_t>;
+
 void ngfunction_2_irv10(pugi::xml_node& node,
                         std::ostream& bin_file,
```

Comment:
`bin_file` should be transparent how we wrote data. Consider it like a class where you may switch on and off compression.
It should provide compression mechanism by itself.

Reply:
Done, let me know what you think

Reply:
ok